tests/remote/remote_controller.pm

```diff
@@ -17,39 +17,51 @@ use testapi;
 use utils;
 use mm_network;
 use lockapi;
+use mmapi;
 
 sub run {
-    my ($self) = @_;
     my $target_ip;
+    my $children = get_children();
+    my $child_id = (keys %$children)[0];
+    my $password = 'nots3cr3t';
 
-    select_console 'root-console';
-    ensure_serialdev_permissions;
-
-    # Setup static NETWORK
-    configure_default_gateway;
-    configure_static_ip('10.0.2.12/15');
+    mutex_lock("dhcp_client_configuration", $child_id);
+    mutex_unlock("dhcp_client_configuration", $child_id);
+    mutex_lock("ssh_server_is_available", $child_id);
 
     # Wait until target becomes ready
-    mutex_lock "installation_ready";
+    # parse dhcpd.leases - now it should contain entries for all nodes
```

Comment:
You can make it easier by setting EXTRABOOTPARAMS=hostname=XXX and then connecting to XXX.openqa.test - but that's up to you ;)